src/mmw/apps/modeling/views.py

```diff
@@ -557,7 +557,8 @@ def start_celery_job(task_list, job_input, user=None,
     """
     created = now()
     job = Job.objects.create(created_at=created, result='', error='',
-                             traceback='', user=user, status='started')
+                             traceback='', user=user, status='started',
+                             model_input=job_input)
```

Comment:
Adding `model_input` on the job here so it's saved with this task:
```
error = save_job_error.s(job.id).set(exchange=exchange,
                                         routing_key=routing_key)
```

This introduces a little redundancy at our call for `save_job_result(job.id, job_input)`. I think it's better for now to make this change as targeted as possible, and not introduce the more wide-reaching change of removing the `model_input` param of `save_job_result`...we use the function a lot throughout the project, so when I had changed the parameters, it was breaking tests